### PR TITLE
Fix build of Alfresco Solr enterprise images on Xenit's build server

### PR DESCRIPTION
**Pull request by koenvangeerteruy**
Hello @thijslemmens  ,

Due to the cleanup of init.gradle on Xenit's build server, the Alfresco Solr enterprise images don't build anymore.

With the updates on docker-openjdk and docker-tomcat, I think it is a good idea to rebuild the docker-solr images so they contain the last openjdk and tomcat updates.

Question regarding all the branches in this project
Can I clean up (delete) all the branches that have been merged or closed?

Kind regards,
Koen

### CODE REVIEW

File: build.gradle [hunk 1]
```diff
@@ -14,10 +14,21 @@ static def calcTags (version) {
 subprojects {
     repositories {
         mavenCentral()
-        jcenter()
         maven {
             url 'https://artifacts.alfresco.com/nexus/content/repositories/public/'
         }
+        // This private repository provides Xenit with Alfresco enterprise artefacts.
```
Comment (thijslemmens) [inline build.gradle:20]: Shouldn't we add Alfresco's enterprise repo instead?

**Comment by thijslemmens** [comment]
> Hello @thijslemmens ,
> 
> Due to the cleanup of init.gradle on Xenit's build server, the Alfresco Solr enterprise images don't build anymore.
> 
> With the updates on docker-openjdk and docker-tomcat, I think it is a good idea to rebuild the docker-solr images so they contain the last openjdk and tomcat updates.
> 
> Question regarding all the branches in this project
> Can I clean up (delete) all the branches that have been merged or closed?
> 
> Kind regards,
> Koen

Yes we should delete merged branches

**Comment by koenvangeerteruy** [comment]
> > Hello @thijslemmens ,
> > Due to the cleanup of init.gradle on Xenit's build server, the Alfresco Solr enterprise images don't build anymore.
> > With the updates on docker-openjdk and docker-tomcat, I think it is a good idea to rebuild the docker-solr images so they contain the last openjdk and tomcat updates.
> > Question regarding all the branches in this project
> > Can I clean up (delete) all the branches that have been merged or closed?
> > Kind regards,
> > Koen
> 
> Yes we should delete merged branches

All merged branches have been deleted.
I was not sure whether the closed PR branch (not merged) could also be deleted.
Current open PR branches and branches without PRs will not be deleted.

File: build.gradle (hunk 1 above)
Comment (koenvangeerteruy) [reply build.gradle:20]: Yes, that is also a possibility. I believe we had also credential settings for Alfresco's Nexus enterprise access in the gradle.properties.
I think we should align with @tgeens what approach he prefers.

**Review by vierbergenlars — APPROVED** [review]

**Pull request merged**